Pokedex/src/resources/resources.ts

```diff
@@ -5,3 +5,10 @@ export const SEARCH_BUTTON = "Search";
 export const CLEAR_BUTTON = "Clear";
 export const RECENT_SEARCHES_BUTTON = "Recent Searches";
 
+export const MAP_ID = "51395591ef744d0";
+
+export const DRIVING = "DRIVING";
+export const TRANSIT = "TRANSIT";
+export const WALKING = "WALKING";
+export const BICYCLING = "BICYCLING";
+
```

Comment:
the resources should be under the same JSON. 

for example: 

const RESOURCES = {
DRIVING: "DRIVING"
}